spring-rest/src/main/java/net/baeldung/ApplicationMain.java

```diff
@@ -0,0 +1,20 @@
+package net.baeldung;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.builder.SpringApplicationBuilder;
+import org.springframework.boot.web.support.SpringBootServletInitializer;
+
+@SpringBootApplication
+public class ApplicationMain extends SpringBootServletInitializer {
```

Comment:
Do we need SpringBootServletInitializer? I believe that should work only with the @SpringBootApplication